content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -106,3 +106,26 @@ Starting from `v1.5.0`, the following deprecated custom resource fields will be
 - BackingImageManager.status.backingImageFileMap.directory
 - BackingImageManager.status.backingImageFileMap.downloadProgress
 - BackingImageManager.status.backingImageFileMap.url
+
+### Engine Upgrade Enforcement
+
+Starting from `v1.6.0`, Longhorn will implement mandatory version upgrades for the engines. The minimum required supporting version will be outlined in the [release note](https://github.com/longhorn/longhorn/releases/tag/v{{< current-version >}}).
```

Comment:
```suggestion
Beginning with version v1.6.0, Longhorn is implementing mandatory engine upgrades. See the [release note]([https://github.com/longhorn/longhorn/releases/tag/v{{](https://github.com/longhorn/longhorn/releases/tag/v%7B%7B)< current-version >}}) for information about the minimum supported engine image version.
```

Reply:
![Screenshot from 2023-10-18 16-59-04](https://github.com/longhorn/website/assets/22583541/0cae3e38-6748-412c-b871-2ac6931d7c4b)

The format is a bit off, but I've updated your suggested words.

Reply:
I made a mistake in copying/pasting the URL that resulted in extra characters. Sorry about that.